src/PagePaginationButtons.jsx

```diff
@@ -0,0 +1,47 @@
+import { Pagination } from 'rsuite';
+import { FlexboxGrid } from 'rsuite';
+import FlexboxGridItem from 'rsuite/lib/FlexboxGrid/FlexboxGridItem';
```

Comment:
what the weird path to the component??

Reply:
it wasnt my
this line was created automatically

Reply:
done